overlay.py

```diff
@@ -81,6 +85,11 @@ def publish_video_status(self, message: str) -> None:
         )
         self.client.publish(online_status_topic, message, retain=True)
 
+    def publish_camera_status(self, message: str) -> None:
+        """ Send a message on the current device's camera status topic. """
+        status_topic = f"/v3/status/camera/{self.device}"
```

Comment:
This will get changed to use `DAShboard` after [common#9](https://github.com/monash-human-power/common/pull/9) gets merged.